Tasks/PerformActionOnMachineGroup/Helper.ps1

```diff
@@ -23,13 +23,16 @@ function Invoke-OperationHelper
 
     Foreach($machine in $machines)
     {
+        $machineName = $machine.Name
         $operation = Invoke-OperationOnProvider -machineGroupName $machineGroupName -machineName $machine.Name -operationName $operationName
+        Write-Verbose "[Azure Resource Manager]Call to provider to perform operation '$operationName' on the machine '$machineName' completed" -Verbose
```

Comment:
[Azure Resource Manager]Completed the operation $operationName on the machine $machineName?